synapse/tests/test_lib_scope.py

```diff
@@ -4,7 +4,8 @@
 
 class ScopeTest(s_t_utils.SynTest):
 
-    def test_lib_scope(self):
+    async def test_lib_scope(self):
```

Comment:
This uses a local scope object. Why is it async?

Reply:
because scope is now async *task* scope, not thread scope.